commands/upload.rb

```diff
@@ -0,0 +1,50 @@
+require 'uri'
+require 'digest/sha2'
+require 'net/http'
+require_relative '../lib/color'
+require_relative '../lib/const'
+
+class Command
+  def self.upload(pkg)
+    # Fetch the GITLAB_TOKEN environment variable and set it to gitlab_token, aborting if it is not set.
+    gitlab_token = ENV.fetch('GITLAB_TOKEN') { abort 'GITLAB_TOKEN environment variable not set.'.lightred }
+
+    # If the package has the no_zstd property, the binaries will be compressed as .tar.xz
+    # Otherwise, the binaries will be compressed as .tar.zst
+    binary_compression = pkg.no_zstd? ? 'tar.xz' : 'tar.zst'
+
+    # Get the name of the compressed binary.
+    pkg_tarfile = "#{pkg.name}-#{pkg.version}-chromeos-#{ARCH}.#{binary_compression}"
+    # Get the absolute path to the compressed binary, aborting if there is no file there.
+    abort "No compiled binary found for #{pkg.name}.".lightred unless File.file?(pkg_tarfile_path = File.join(CREW_LOCAL_BUILD_DIR, pkg_tarfile))
```

Comment:
The `File.file?` parameter is an assignment, not a file.

Reply:
It both assigns the value and checks that the file exists.